randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -222,23 +225,23 @@ Extra - asset_id: 4
 > Door to Attic; Heals? False
   * Layers: default
   * Normal Door to Attic/Door to West Ocean
-  > Door to Bowling Alley Cave
-      All of the following:
-          Morph Ball
-          Morph Ball Bombs or Power Bombs or Spring Ball
   > Door to Bowling Alley
       Trivial
   > Pickup (Topmost Missile Expansion)
       Any of the following:
-          # Knowledge (Intermediate): Trippers can be brought over to right platforms by keeping them on screen. (jump and aim up)
-          Hi-Jump Boots or Space Jump or Knowledge (Intermediate)
+          Hi-Jump Boots or Space Jump
+          # Shinespark
           Speed Booster and Shinespark (Beginner) and Normal Damage ≥ 36
           # Wall jump off of Trippers
           Ice Beam and Wall Jumps (Intermediate)
           # IBJ
           Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
+          # Knowledge (Intermediate): Trippers can be brought over to right platforms by keeping them on screen. (jump and aim up)
+          Knowledge (Intermediate)
+          # Speedboost and jump. TODO: better video - https://youtu.be/t5Va2J2zmiQ
+          Speed Booster and Movement (Beginner)
```

Comment:
This also requires HJ. Here is another video https://youtu.be/rkw3gUtLFqQ

Reply:
Thanks for spontanously recording all of those vids